docs/compilers/CSharp/Compiler Breaking Changes - VS2019.md

```diff
@@ -81,3 +81,26 @@ Each entry should include a short description of the break, followed by either a
 
 10. Previously, reference assemblies were emitted including embedded resources. In Visual Studio 2019, embedded resources are no longer emitted into ref assemblies.
   See https://github.com/dotnet/roslyn/issues/31197
+
+11. Ref structs now support disposal via pattern. A ref struct enumerator with an accessible `void Dispose()` method will now have it invoked at the end of enumeration, where it would not have been before:
```

Comment:
>method [](start = 110, length = 6)

"... instance method or static extension method"

Reply:
LDM disallowed extension methods. (I *really* need to publish the updated spec)

---
In reply to: [256627330](https://github.com/dotnet/roslyn/pull/33367#discussion_r256627330) [](ancestors = 256627330)